src/main/java/com/google/sps/utils/UserUtils.java

```diff
@@ -47,25 +47,43 @@
     * @param  entityPropertyValue  the value of the property that is filtered for
     * @return                      a single entity that has the same value for the property in the parameter 
     */
```

Comment:
You've added a new function right between the documentation for `getEntityFromDatastore` and the definition of `getEntityFromDatastore`. Please move the new code somewhere else, and give it its own documentation.